example/satosa/pyeudiw_backend.yaml

```
@@ -19,11 +19,10 @@ config:
       get_response: '/get-response'
 
   qrcode:
-    size: 100
-    color: '#2B4375'
+    size: 250 # px
+    color: '#000000'  # hex
     expiration_time: 120 # seconds
-    logo_path: 
-    use_zlib: false
+    logo_path: '/static/wallet-it/wallet-icon-blue.svg' # relative to static_storage_url
```

```suggestion
    logo_path: '/wallet-it/wallet-icon-blue.svg' # relative to static_storage_url
```

because `/static` is,  must be, part of the configured static url in the configuration